src/components/AssetInfo/TokenSideBarField.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./TokenSideBar.scss";
+
+const TokenSideBarField = (props: {
+  title: string;
+  ctaText: string;
+  ctaStatus?: string;
+  children?: React.ReactNode;
+  isChangeBeneficiary?: boolean;
+}) => {
+  let buttonCss = `${styles["button"]}`;
+  if (props.ctaStatus === "success") {
+    buttonCss = `${styles["button"]} ${styles["button-success"]}`;
+  } else if (props.ctaStatus === "danger") {
+    buttonCss = `${styles["button"]} ${styles["button-danger"]}`;
+  } else {
+  }
+
```

Comment:
<button className={`${styles.button} ${props.ctaStatus === "success" ? styles["button-success"]: styles["button-danger"]}`}>{props.ctaText}</button>

replace with this

Reply:
I prefer not to have ternary condition for this part... to maintain readability and also in case there are more button variation styles in the future.